src/sas/system/web.py

```diff
@@ -0,0 +1,24 @@
+class WebLinks:
+    def __init__(self):
+        self.nist_url = "https://www.nist.gov/ncnr"
+        self.umd_url = "https://www.umd.edu/"
+        self.sns_url = "https://neutrons.ornl.gov/"
+        self.nsf_url = "https://www.nsf.gov"
+        self.isis_url = "https://www.isis.stfc.ac.uk/"
+        self.ess_url = "https://europeanspallationsource.se/"
+        self.ill_url = "https://www.ill.eu/"
+        self.ansto_url = "https://www.ansto.gov.au/"
+        self.bam_url = "http://www.bam.de/"
+        self.delft_url = "http://www.tudelft.nl/en/tnw/business/facilities/reactor-instituut-delft/"
```

Comment:
As noted by @smk78 in #2200:

> All the links from the About box work, but I notice that TU Delft have made a change so that URL should now redirect to:
https://www.tudelft.nl/en/faculty-of-applied-sciences/business/facilities/reactor-institute-delft